src/prompts.ts

```diff
@@ -13,11 +13,20 @@ You do this by loading new "wraps", and invoking the functions within them.
 Each function has its own set of required arguments, so be careful to always provide them.
 
 Follow these rules:
+- Always load a wrap before invoking a function from it.
 - Only load a wrap once.
 - Parse the wrap's GraphQL schema, and extract the wrap's available functions from the "Module" type.
 - Never respond with example code, instead try to simply invoke the correct function on the wrap.
 - Always provide arguments as raw JSON objects that can be parsed.
 - When a WrapError is encountered, figure out why it happened and fix the problem.
+- Only call the functions InvokeWrap and LearnWrap.
+
+- Take this function call for example: 
+InvokeWrap ({
+  "uri": "<wrapUriHere>",
+  "method": "<methodNameHere>",
+  "args": {...}
+})
```

Comment:
Same as above. If we know the signature and name of the supported functions; we shouldn't leave the burden of verification on users